### ISSUE TITLE
YouTube is broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MiraculousLegendas/get-flash-videos#291]
```
./get_flash_videos -d "http://www.youtube.com/watch?v=a1Y73sPHKxw"
```

Original issue reported on code.google.com by `richcoll...@gmail.com` on 4 Aug 2011 at 11:44
- Merged into: #288

**Comment by GoogleCodeExporter** [comment]
```
Priority: HIGH!
```

Original comment by `seegahan@gmail.com` on 6 Aug 2011 at 6:23

**Comment by GoogleCodeExporter** [comment]
```
Fixed by richcollins in commit 27e0f1d.
```

Original comment by `mjbauer95` on 7 Aug 2011 at 5:12
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `mjbauer95` on 7 Aug 2011 at 5:14
- Changed state: **Duplicate**
